recipes/oommf/build.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+TCLTKVERSION=8.5
+CONDA_ROOT_PATH=$(conda info --root)
+
+# Compile OOMMF
+export OOMMF_TCL_CONFIG=${PREFIX}/lib/tclConfig.sh
+export OOMMF_TK_CONFIG=${PREFIX}/lib/tkConfig.sh
+cd oommf
+tclsh${TCLTKVERSION} oommf.tcl pimake distclean
```

Comment:
Thanks. There is currently some confusion over tclsh/tk8.5 and tclsh/tk8.6 being build differently (with threading enabled or without), so I am not sure this is actually the same. Is there any update on this, or recommendation on how to proceed here, @takluyver ?